README.md

```diff
@@ -113,6 +113,14 @@ In your application you will need to do the following:
 * Replace references to `id7-bundle.min.js` with `id7-bundle.js`
 * `id7-standalone.js` was removed from the distribution; you should build your own bundle if you are using this
 
+## Using ID7.1
+
+From version 2.9.7 onwards, you can apply the new "ID7.1" styling to the site masthead in your app. To do so:
+
+* Add the `id7-point-1` class to any `id7-utility-masthead` elements in your templates
+* Replace the logo image src `/dist/images/logo.png` with `/dist/images/logo.svg` in your templates
```

Comment:
* It won't always be `/dist/images`, that's just where the demo site serves it from
* A bunch of our existing templates seem to use an empty shim.gif instead of logo.png, so it might be better to refer to this by selector e.g. `.id7-logo > a > img`

Reply:
Updated